cactus/bootstrap/package.py

```diff
@@ -1,16 +1,16 @@
-#coding:utf-8
+# coding:utf-8
+from cactus.cli import PATH
```

Comment:
The constant "PATH" uses only in this file and not uses in cactus.cli. And cactus.cli it's top the top level module We have no reason to know about cactus.cli in bootstrap/package module.

I think we better way is to remove PATH in cactus.cli and put it into /bootstrap/package

Reply:
i cant move PATH to /bootstrap/package due to importError, i can move it to utils/packaging

Reply:
utils/packaging will be good. And delete in cli.py